tests/datasets/test_dataset.py

```diff
@@ -18,13 +18,19 @@
 from __future__ import annotations
 
 import os
+from collections import defaultdict
 
 import pytest
+from sqlalchemy.sql import select
 
 from airflow.datasets import Dataset
+from airflow.models.dataset import DatasetAll, DatasetAny, DatasetDagRunQueue, DatasetModel
+from airflow.models.serialized_dag import SerializedDagModel
 from airflow.operators.empty import EmptyOperator
+from airflow.serialization.serialized_objects import BaseSerialization, SerializedDAG
 
 
+@pytest.mark.db_test
```

Comment:
Why do all of these need to db tests now?

Reply:
I am not sure, @sunank200 added that in response to test failures i can take a look.

Reply:
oh yeah it's cus an autouse fixture was added
```
@pytest.fixture(autouse=True)
def clear_datasets():
    from tests.test_utils.db import clear_db_datasets

    clear_db_datasets()
```

i'll remove that

Reply:
That's true its for `clear_datasets()`

Reply:
Gotcha. Let's be more targeted with what tests need to hit the db and which don't. db tests aren't as quick.

Reply:
fixed it